src/test/java/newbank/TestApp.java

```diff
@@ -195,4 +195,11 @@ public void canHandleInvalidDepositAccountOrAmount() throws IOException {
     response = testCommand("DEPOSIT Savings -500.0\n");
     assertThat(response, containsString("FAIL"));
   }
+
+  @Test
+  public void canExitApplication() throws IOException, InterruptedException {
+    String response = testCommand("QUIT\n");
+    assertThat(response, containsString("SUCCESS"));
```

Comment:
Curious if we could (easily) assert that the thread has terminated too.

Reply:
This is what the "client.join()" does.